edge-modules/TwinTester/DesiredPropertiesValidator.cs

```diff
@@ -11,55 +11,25 @@ namespace TwinTester
     using Microsoft.Extensions.Logging;
     using Newtonsoft.Json;
 
-    class DesiredPropertyOperation : TwinOperationBase
+    class DesiredPropertiesValidator : ITwinPropertiesValidator
     {
-        static readonly ILogger LoggerImpl = ModuleUtil.CreateLogger(nameof(DesiredPropertyOperation));
+        static readonly ILogger Logger = ModuleUtil.CreateLogger(nameof(DesiredPropertiesValidator));
         readonly RegistryManager registryManager;
+        readonly TwinState twinState;
         readonly ModuleClient moduleClient;
-        readonly AnalyzerClient analyzerClient;
         readonly TwinEventStorage storage;
-        readonly TwinState twinState;
+        readonly IResultHandler resultHandler;
 
-        public DesiredPropertyOperation(RegistryManager registryManager, ModuleClient moduleClient, AnalyzerClient analyzerClient, TwinEventStorage storage, TwinState twinState)
+        public DesiredPropertiesValidator(RegistryManager registryManager, ModuleClient moduleClient, TwinEventStorage storage, IResultHandler resultHandler, TwinState twinState)
         {
             this.registryManager = registryManager;
```

Comment:
Do we need to validate these inputs?

Reply:
This class is internal so I don't think we need to validate inputs

---
In reply to: [357865085](https://github.com/Azure/iotedge/pull/2108#discussion_r357865085) [](ancestors = 357865085)